### PR TITLE
Fix delimiter detection with quoted cells #1855

### PR DESCRIPTION
**Pull request by rafatower**
@Kartones review please

### CODE REVIEW

File: services/importer/spec/unit/csv_normalizer_spec.rb [hunk 1]
```diff
@@ -194,6 +200,19 @@ def single_column_factory
     filepath
   end
 
+  def quoted_string_with_delimiter_factory
+    filepath = get_temp_csv_fullpath
+
+    ::File.open(filepath, 'w') do |file|
+      file << 'name,description ; with semicolon,wadus' << "\n"
+      file << 'foo,"this description contains; a semicolon and a, comma to affect frequency table",bar' << "\n"
+      file << 'foobar,"this description contains; a semicolon but no comma",barfoo' << "\n"
```
Comment (Kartones) [inline services/importer/spec/unit/csv_normalizer_spec.rb]: Can you test if this breaks? `"123","this is valid \" aaa"`, "field3"

Comment (rafatower) [inline services/importer/spec/unit/csv_normalizer_spec.rb]: You're right, I'll look for a better solution (and add that test). Thanks!

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2100/))

File: services/importer/spec/unit/csv_normalizer_spec.rb (hunk 1 above)
Comment (rafatower) [inline services/importer/spec/unit/csv_normalizer_spec.rb]: Can you please take another look? I know it is not perfect but that should be good enough for detection. All ears about your opinion

**Comment by Kartones** [comment]
+1

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2106/))

**Pull request merged**